test/integration/webhook/v1alpha2/clusterqueue_test.go

```diff
@@ -61,10 +68,33 @@ var _ = ginkgo.Describe("ClusterQueue Webhook", func() {
 			gomega.Expect(created.GetFinalizers()).Should(gomega.Equal([]string{kueue.ResourceInUseFinalizerName}))
 		})
 
+		ginkgo.It("Should have qualified clusterQueueingStrategy when creating", func() {
```

Comment:
Can you make it a `DescribeTable` too? along with other creation validations.


And perhaps a separate `DescribeTable` for all update validtions, if more than one.

Reply:
Wouldn't it be better to fix this in a new PR?

Reply:
Have done in this pr.